### PR TITLE
Add Ruby to Template Documentation

### PR DESCRIPTION
**Pull request by affix**
Signed-off-by: Keiran Smith <contact@keiran.scot>

Adds ruby guide to the cli/templates documentation

requested by @alexellis

### CODE REVIEW

File: docs/cli/templates.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ pycon/requirements.txt
 
 ### 2.1 Python: dependencies
 
-You should edit `pycon/requirements.txt` and add any pip modules you want with each one on a new line, for instance `requests`. 
+You should edit `pycon/requirements.txt` and add any pip modules you want with each one on a new line, for instance `requests`.
```
Comment (affix) [inline docs/cli/templates.md]: fixed an extra whitespace

File: docs/cli/templates.md [hunk 2]
```diff
@@ -145,11 +145,86 @@ Now you can open your current folder in a tool such as Visual Studio Code and ad
...
+Your code should be in the handler.rb file
+
+### 5.1 Adding a Gem (Library)
+
+Open the `Gemfile` in the ruby-function directory
+
+Add the following line
+
+```
+gem 'qrcoder'
```
Comment (alexellis) [inline docs/cli/templates.md]: This should be `httparty` I think?

**Comment by alexellis** [comment]
Nice :+1: I think I see one minor typo.

File: docs/cli/templates.md (hunk 2 above)
Comment (affix) [reply docs/cli/templates.md]: I fixed that before you submitted your comment haha

**Pull request merged**